src/components/elementView/elementTable.tsx

```diff
@@ -800,9 +804,12 @@ class ElementTable extends React.Component<
               {this.state.items && this.state.items.length > 0 ? (
                 <tbody>
                   {this.state.items.map((item: ElementObjectType) => {
+                    if (isAssetObject(item)) {
+                      console.log(item)
+                    }
                     return (
                       <tr
-                        onClick={() => {
+                        onClick={this.props.type === "datacenters" ? () => { } : () => {
```

Comment:
Can you use the enum just for consistency? `ElementTypes.DATACENTER`. Also while you're at it, can you disable clicking for users as well?